src/Life/Main/Remove.hs

```diff
@@ -0,0 +1,62 @@
+{-# LANGUAGE Rank2Types       #-}
+{-# LANGUAGE TypeApplications #-}
+
+-- | Functions to remove from your life.
+
+module Life.Main.Remove
```

Comment:
I see now then my `Update` name I gave to module was an oversee. Could you also plese remove `Update` module to `Add`?